### PR TITLE
Add the vendor, summary, install time and size properties from the RP…

### PR DESCRIPTION
**Pull request by NiuBlibing**
…M database Generate updated test cases using a shell script

Signed-off-by: Tim Ferguson <drtimf@gmail.com>
Signed-off-by: Tianyue Ren <rentianyue-jk@360shuke.com>

Issue [#16](https://github.com/knqyf263/go-rpmdb/issues/16)

*Description of changes:*

By submitting this pull request, I confirm that you can use, modify, copy, and redistribute this contribution, under the terms of your choice.

### CODE REVIEW

File: pkg/rpmdb_testcase.go [hunk 1]
```diff
@@ -2,387 +2,387 @@ package rpmdb
...
-		{0, "libXdamage", "1.1.3", "4.el6", "x86_64", "libXdamage-1.1.3-4.el6.src.rpm", 24896, "MIT", "CentOS", "", nil, nil, nil},
-		{0, "libXinerama", "1.1.3", "2.1.el6", "x86_64", "libXinerama-1.1.3-2.1.el6.src.rpm", 12039, "MIT", "CentOS", "", nil, nil, nil},
-		{0, "pango", "1.28.1", "11.el6", "x86_64", "pango-1.28.1-11.el6.src.rpm", 1041857, "LGPLv2+", "CentOS", "", nil, nil, nil},
-		{0, "mailcap", "2.1.31", "2.el6", "noarch", "mailcap-2.1.31-2.el6.src.rpm", 52877, "Public Domain and MIT", "CentOS", "", nil, nil, nil},
-		{0, "perl-XML-Parser", "2.36", "7.el6", "x86_64", "perl-XML-Parser-2.36-7.el6.src.rpm", 652307, "GPL+ or Artistic", "CentOS", "", nil, nil, nil},
-		{0, "policycoreutils", "2.0.83", "30.1.el6_8", "x86_64", "policycoreutils-2.0.83-30.1.el6_8.src.rpm", 3596110, "GPLv2+", "CentOS", "", nil, nil, nil},
+		{0, "policycoreutils", "2.0.83", "30.1.el6_8", "x86_64", "policycoreutils-2.0.83-30.1.el6_8.src.rpm", 3596110, "GPLv2+", "CentOS", "SELinux policy core utilities", 1647005398, "", nil, nil, nil},
```
Comment (knqyf263) [inline pkg/rpmdb_testcase.go:123]: Do you know why the order is updated?

File: pkg/gen_testcase.sh [hunk 2]
```diff
@@ -0,0 +1,77 @@
...
+}
+
+cat << EOT
+package rpmdb
+
+var (
+	// docker run --rm -it centos:6 bash
+	// rpm -qa --queryformat "\{%{EPOCH}, \"%{NAME}\", \"%{VERSION}\", \"%{RELEASE}\", \"%{ARCH}\", \"%{VENDOR}\", \"%{SUMMARY}\", %{SIZE}, %{INSTALLTIME}\},\n" | sed "s/(none)/0/g"
+	CentOS6Plain = []PackageInfo{
+`getListing "Packages_centos6_plain"`
```
Comment (knqyf263) [inline pkg/gen_testcase.sh:19]: How does it work? The file path doesn't exist.

**Review by knqyf263 — COMMENTED** [review]
Thanks. I left some comments.

File: pkg/gen_testcase.sh (hunk 2 above)
Comment (NiuBlibing) [reply pkg/gen_testcase.sh]: Sorry,I forgot to refine test cases,the second patch fixed it.

File: pkg/rpmdb_testcase.go (hunk 1 above)
Comment (NiuBlibing) [reply pkg/rpmdb_testcase.go]: Maybe because that I regenerated the package database.

Comment (knqyf263) [reply pkg/rpmdb_testcase.go]: Hmm. Why did you regenerate the package database? I saw many changes and it makes it hard to confirm the change is good.

Comment (NiuBlibing) [reply pkg/rpmdb_testcase.go]: I did not see the test package database before, and then I saw them when I finished the patch.I'll generate a new patch later.

File: pkg/gen_testcase.sh (hunk 2 above)
Comment (NiuBlibing) [reply pkg/gen_testcase.sh]: @knqyf263 I've create a new [pr](https://github.com/knqyf263/go-rpmdb/pull/18).

**Comment by NiuBlibing** [comment]
Replaced with [PR18](https://github.com/knqyf263/go-rpmdb/pull/18)

**Pull request closed without merging**